src/main/java/ledger/user_interface/ui_controllers/MainPageController.java

```diff
@@ -357,10 +357,10 @@ private void handleDeleteTransactionFromTableView() {
                 task.startTask();
                 task.waitForComplete();
             } else {
-                System.out.println("No transactions deleted. Selected index: " + selectedIndex);
+                setupErrorPopup("No transactions deleted. Selected index.");
             }
         } catch (StorageException e) {
-            System.out.println("Error deleting transaction: " + e.getMessage());
+            setupErrorPopup("Error deleting transaction.");
```

Comment:
Should  probably keep the message. Latter definitely want to translate it but even the bare message should be useful.